src/enums/level.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum Level {
+  easy = 'easy',
```

Comment:
i think its better to save level as a number 1, 2, 3, 4 
its much smaller in the DB